src/Microsoft.AspNet.Mvc.Core/Rendering/Internal/TemplateRenderer.cs

```diff
@@ -7,16 +7,19 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using Microsoft.AspNet.Http;
 using Microsoft.AspNet.Mvc.Core;
+using Microsoft.AspNet.Mvc.ModelBinding;
 using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Internal;
 
-namespace Microsoft.AspNet.Mvc.Rendering
+namespace Microsoft.AspNet.Mvc.Rendering.Internal
 {
-    internal class TemplateRenderer
+    public class TemplateRenderer
     {
         private static readonly string DisplayTemplateViewPath = "DisplayTemplates";
         private static readonly string EditorTemplateViewPath = "EditorTemplates";
+        public static readonly string IEnumerableOfIFormFileName = "IEnumerable`" + nameof(IFormFile);
```

Comment:
Oh I think that might be because it was erroneously assumed that this is public, whereas it really isn't. This member is effectively internal (it used to be C# internal but now it's in the Internal namespace) so we can use const's all we want.

@NTaylorMullen @dougbu did I miss anything here on the reasoning?